data/cncf.yaml

```diff
@@ -3426,3 +3428,17 @@
       check_sets:
         - community
         - code
+- name: kueue
+  display_name: kueue
+  description: Kubernetes-native Job Queueing.
+  category: runtime
+  logo_url: https://raw.githubusercontent.com/kubernetes-sigs/kueue/main/site/static/images/logo.svg
+  devstats_url: https://kueue.devstats.cncf.io/
```

Comment:
Is this page going to be live once this PR merges?

Reply:
see example https://kcl.devstats.cncf.io/